web/ui/react-app/src/types/types.ts

```diff
@@ -0,0 +1,22 @@
+export type Nullable<T> = T | null;
+
+export interface GraphSeries {
```

Comment:
I generally wonder what the motivation is to move some of these types to a different file - as long as they are only used in one file (Graph.tsx), wouldn't they be better / easier kept and maintained there?

Reply:
Yes, I'm kind of experimenting. I was wonder whether would it not be better if we move those interfaces (not Props and State) in dedicated file. Thus we will have a better look what we have as interfaces and components will shrink a bit. WDYT?

Reply:
Generally I think I'm a fan of keeping types local to a component if they are only used by the component, and only separate them out when they are really shared.